main.tf

```diff
@@ -1,3 +1,280 @@
+  count      = var.create_ws_liberty_operator_namespace == true ? 1 : 0
+
+  metadata {
+    name = var.ws_liberty_operator_namespace
+  }
+
+  timeouts {
+    delete = "30m"
+  }
+
+  lifecycle {
+    ignore_changes = [
+      metadata[0].annotations,
+      metadata[0].labels
+    ]
+  }
+}
+
+resource "helm_release" "websphere_liberty_operator" {
+  depends_on = [time_sleep.wait_catalog[0], kubernetes_namespace.websphere_liberty_operator_namespace[0]]
+
+  name              = "websphere-liberty-operator-helm-release"
+  chart             = "${path.module}/chart/${local.websphere_liberty_operator_chart}"
+  namespace         = var.operator_helm_release_namespace
+  create_namespace  = false
+  timeout           = 300
+  dependency_update = true
+  force_update      = false
+  cleanup_on_fail   = false
+  wait              = true
+
+  disable_openapi_validation = false
+
+  set {
+    name  = "operatornamespace"
+    type  = "string"
+    value = var.ws_liberty_operator_namespace
+  }
+
+  provisioner "local-exec" {
+    command     = "${path.module}/scripts/approve-install-plan.sh ${var.ws_liberty_operator_namespace}"
+    interpreter = ["/bin/bash", "-c"]
+    environment = {
+      KUBECONFIG = data.ibm_container_cluster_config.cluster_config.config_file_path
+    }
+  }
+}
+
+resource "time_sleep" "wait_websphere_liberty_operator" {
+  depends_on = [helm_release.websphere_liberty_operator]
+
+  create_duration = local.sleep_time_operator_create
+}
+
+##############################################################################
+# Confirm websphere operator is operational
+##############################################################################
+
+resource "null_resource" "confirm_websphere_liberty_operator_operational" {
+
+  depends_on = [time_sleep.wait_websphere_liberty_operator]
+
+  provisioner "local-exec" {
+    command     = "${path.module}/scripts/confirm-wsloperator-operational.sh ${var.ws_liberty_operator_namespace}"
+    interpreter = ["/bin/bash", "-c"]
+    environment = {
+      KUBECONFIG = data.ibm_container_cluster_config.cluster_config.config_file_path
+    }
+  }
+}
+
+resource "kubernetes_namespace" "websphere_liberty_sampleapp_namespace" {
+  depends_on = [null_resource.confirm_websphere_liberty_operator_operational]
+  count      = var.install_wslo_sampleapp == true ? 1 : 0
+
+  metadata {
+    name = var.wslo_sampleapp_namespace
+  }
+
+  timeouts {
+    delete = "30m"
+  }
+
+  lifecycle {
+    ignore_changes = [
+      metadata[0].annotations,
+      metadata[0].labels
+    ]
+  }
+}
+
+locals {
+  websphere_liberty_operator_sampleapp_image_path    = "icr.io/appcafe/open-liberty/samples/getting-started"
+  websphere_liberty_operator_sampleapp_image_version = "latest@sha256:d735c2ceae5945a0f20adcbcb04e55472d2520b6d1abb6d3049c8521234d3b7a"
+}
+
+resource "helm_release" "websphere_liberty_operator_sampleapp" {
+  depends_on = [kubernetes_namespace.websphere_liberty_sampleapp_namespace[0]]
```

Comment:
remove depends_on and use `create_namespace = true`

Reply:
same of https://github.com/terraform-ibm-modules/terraform-ibm-websphere-liberty-operator/pull/3#discussion_r1483277366